xrootd/xrootd_config.go

```diff
@@ -313,7 +313,7 @@ func CheckXrootdEnv(server server_utils.XRootDServer) error {
 		}
 	}
 
-	if server.GetServerType().IsSet(config.OriginType) {
+	if server.GetServerType().IsEnabled(config.OriginType) {
```

Comment:
Here is one last place where we want to differentiate between "enabled" vs. "calling" server.

Reply:
Same as the one in `xroot/authorization`, the method override will ensure it behaves as expected for running multiple servers.